src/main/java/io/r2dbc/postgresql/PostgresqlConnectionConfiguration.java

```diff
@@ -285,6 +360,83 @@ public Builder schema(@Nullable String schema) {
             return this;
         }
 
+        /**
+         * Configure ssl usage.
+         *
+         * @param ssl
+         * @return this {@link Builder}
+         */
+        public Builder ssl(boolean ssl) {
+            this.ssl = ssl;
+            return this;
+        }
+
+        /**
+         * Configure ssl cert.
+         *
+         * @param sslCert
+         * @return this {@link Builder}
+         */
+        public Builder sslCert(@Nullable String sslCert) {
+            this.sslCert = sslCert;
+            return this;
+        }
+
+        /**
+         * Configure ssl HostnameVerifier.
+         *
+         * @param sslHostnameVerifier
+         * @return this {@link Builder}
+         */
+        public Builder sslHostnameVerifier(@Nullable String sslHostnameVerifier) {
+            this.sslHostnameVerifier = sslHostnameVerifier;
+            return this;
+        }
+
+        /**
+         * Configure ssl key.
+         *
+         * @param sslKey
+         * @return this {@link Builder}
+         */
+        public Builder sslKey(@Nullable String sslKey) {
```

Comment:
Nit: Let's document that this should be a path to the SSL key in PEM format.

Reply:
I agree that ssl parameter is redundant, I've just followed pgjdbc configurations like you said before and it has both.